nni/compression/pytorch/compressor.py

```diff
@@ -720,11 +720,11 @@ def quant_backward(tensor, grad_output, quant_type, scale, zero_point, qmin, qma
         return grad_output
 
     @staticmethod
-    def forward(ctx, tensor, quant_type, wrapper, **kwargs):
+    def forward(ctx, tensor, quant_type, wrapper, tensor_alt=None, **kwargs):
```

Comment:
what is the meaning of `tensor_alt`?

Reply:
It is used to transfer the second tensor during the forward process. In previous implementation, we only transfer one tensor like weight, input, output. But in some situation, we need to transfer two tensors to calibrate both of them like the function `quantize_weight`. So I add it here.

Reply:
is `tensor_alt` commonly used in different quantizers? if it is specific for some quantizers, suggest to put it in `kwargs`, this is how`kwargs` used for

Reply:
The argument `tensor_alt` may not be used by most of quantizers but I don't think it is a bad nothing to provide an alternative preparation here. What's more, `forward` in class `QuantGrad` is called by **apply()** which only supports positional argument so that `kwargs` may be nothing here. If we put it as `kwargs` forcibly, error would be raised. Based on the above reasons, I think it can be kept.